specs/language/placeholders.tex

```diff
@@ -4,12 +4,6 @@
 % forward references can be inserted into the specification as it is being
 % written to force updating the references as they change.
 
-\Ch{Statements}{Stmt}
-\Sec{Label Statements}{Stmt.Label}
```

Comment:
Wondering why Statements was pulled out into its own file, but Classes wasn't?  I can see why Declarations was because this change adds some real content to that.

Reply:
I think @hekota did that just to preserve the chapter ordering, which seems fine to me.

Reply:
Gotcha, that makes sense.